src/flake8_to_ruff/isort.rs

```diff
@@ -0,0 +1,31 @@
+//! Extract isort configuration settings from a pyproject.toml.
+
+use std::path::Path;
+
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+
+// use crate::settings::types::PythonVersion;
```

Comment:
left over comment?

Reply:
Yep, copy-paste from `black.rs` and forgot the remove 👍🏼